keyboards/eniigmakeyboards/ek60/readme.md

```diff
@@ -0,0 +1,9 @@
+# Eniigma Keyboards ek60
+
+* Keyboard Maintainer: [eniigmakeyboards](https://github.com/eniigmakeyboards) and [adamws](https://github.com/adamws)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to reset to the bootloader, to flash new firmware?

Reply:
Done